Sources/IO/Misc/PDBReader/index.js

```diff
@@ -238,7 +190,6 @@ export function extend(publicAPI, model, initialValues = {}) {
     'xSpacing',
```

Comment:
yes [here](https://github.com/Kitware/vtk-js/pull/111/files/eb548f89e24ec17c8ee4a072bd58c6cbee7dfd30#diff-0898b2950a2093dc15347d7b971be6d9R106) but actually we might remove them, not quite useful for now indeed